### PR TITLE
chore: prevent sentry reporting for 4xx errors

### PR DESCRIPTION
**Pull request by usu**

### CODE REVIEW

**Review by carlobeltrame — APPROVED** [review]
Is this integration included with the sentry package? We don't have to install anything new for this?

**Review by BacLuc — APPROVED** [review]

File: api/.env [hunk 1]
```diff
@@ -74,7 +74,7 @@ OAUTH_JUBLADB_BASE_URL="https://jubla.puzzle.ch"
 
 ###> sentry/sentry-symfony ###
 SENTRY_API_DSN=""
-SENTRY_ENVIRONMENT="http://localhost:3000"
+SENTRY_ENVIRONMENT="local"
```
Comment (BacLuc) [inline api/.env:77]: was this change on purpose?

Comment (usu) [reply api/.env:77]: Yes. Sentry doesn't like full URLs as environment

**Comment by usu** [comment]
> Is this integration included with the sentry package? We don't have to install anything new for this?

Yep, already included in the default sentry package

File: api/config/packages/sentry.yaml [hunk 2]
```diff
@@ -3,6 +3,18 @@ sentry:
...
+
+services:
+    Sentry\Integration\IgnoreErrorsIntegration:
+        arguments:
+            $options:
+                ignore_exceptions:
+                    - Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+                    - Symfony\Component\Routing\Exception\ResourceNotFoundException
+                    - Symfony\Component\Security\Core\Exception\AccessDeniedException
+                    - Symfony\Component\HttpKernel\Exception\HttpException
```
Comment (usu) [inline api/config/packages/sentry.yaml:17]: From meeting discussion: Consider using a ValidationException class to trigger 422 errors. This would avoid ignoring exception, if anyone in the future uses HttpException for non 4xx errors
